assets/js/blocks/stock-filter/utils.ts

```diff
@@ -23,8 +23,9 @@ export const getActiveFilters = (
 		? params.split( ',' )
 		: ( params as string[] );
 
-	return Object.keys( filters ).filter( ( filter ) =>
-		parsedParams.includes( filter )
+	const filterKeys = Object.keys( filters );
+
+	return parsedParams.filter( ( param ) => filterKeys.includes( param ) );
 	);
```

Comment:
Just remove this now unnecessary closing round bracket, and we should be all set!
```suggestion
```